RileyLink/TextFieldTableViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TextFieldTableViewController.swift
+//  Naterade
+//
+//  Created by Nathan Racklyeft on 8/30/15.
+//  Copyright © 2015 Nathan Racklyeft. All rights reserved.
+//
+
+import UIKit
+
+protocol TextFieldTableViewControllerDelegate: class {
+    func textFieldTableViewControllerDidEndEditing(controller: TextFieldTableViewController)
+}
+
+
+class TextFieldTableViewController: UITableViewController, IdentifiableClass, UITextFieldDelegate {
```

Comment:
I had been meaning to move this into LoopKitUI, since it's useful boilerplate :)